modules/video/include/opencv2/video/tracking.hpp

```diff
@@ -564,6 +564,12 @@ class CV_EXPORTS_W VariationalRefinement : public DenseOpticalFlow
     /** @copybrief getGamma @see getGamma */
     CV_WRAP virtual void setGamma(float val) = 0;
 
+    /** @brief Param in the norm used by robust penalizer
```

Comment:
"Norm value shift for robust penalizer" sounds better.

Reply:
Sounds good, I will update it asap